src/AboutUs.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+ 
+const AboutUs = (props) => {
```

Comment:
in the src file make a file called pages/profile/ and put all the `Main.js` and at /src/pages/profile/subpages/ put the rest of the pages

Reply:
I'll make that change